src/js/shield_defs.js

```diff
@@ -883,6 +883,19 @@ export function loadShields(shieldImages) {
     Color.shields.white
   );
 
+  // Mexico
+
+  // Carreteras Federales
+  shields["MX:MX"] = {
```

Comment:
I remain a little concerned that `MX:MX` is too easy to confuse with state highways of the State of México (`MX:MEX`). Presumably `MX:MX` was inspired by `US:US`, but we arrived at that value because we literally call them U.S. Routes.

We can land this support for `MX:MX` for now, but stay in touch with the Mexican community in case they flag any concerns as time goes on. If needed, I would suggest two alternatives: either `MX:national`, following the [basic format](https://wiki.openstreetmap.org/wiki/Key:network#Basic_format), or `MX:Fed`, which would be a better analogy to what the other [hierarchical-format countries](https://wiki.openstreetmap.org/wiki/Key:network#Hierarchical_format) came up with. (“Fed.” is a common abbreviation for this network in both English and Spanish.)

Reply:
I note that "Fed" isn't used in way refs but maybe it should be. In any case, with the route relations coming online, it'll at least be easy to change later in a bulk edit if a different community consensus emerges.

Reply:
Yes, just like in the U.S., way `ref` formats have historically been heavily influenced by the needs of pre-Americana renderers, which parse out country and state codes from that value.